src/Augurk.UI/wwwroot/index.html

```diff
@@ -33,6 +33,23 @@
     <script src="lib/underscore.js/underscore-min.min.js"></script>
     <script src="lib/webfont/webfontloader.js"></script>
     <script src="lib/js-sequence-diagrams/dist/sequence-diagram-min.min.js"></script>
+
+    <script lang="javascript">
+        $(function(){
```

Comment:
Perhaps we should add some comments here? Or is this fine?

Reply:
Probably ;).